apis/ops/v1alpha1/mysql_ops_types.go

```diff
@@ -97,6 +102,20 @@ type MySQLHorizontalScalingSpec struct {
 	Member *int32 `json:"member,omitempty"`
 }
 
+type MySQLReplicationModeTransformSpec struct {
+	// Group Replication can be deployed in either "Single-Primary" or "Multi-Primary" mode
+	// +kubebuilder:default=Single-Primary
+	Mode *dbapi.MySQLGroupMode `json:"mode,omitempty"`
```

Comment:
As the main target of this opsRequest is to change the mode,  We should make the `mode` field compulsory for users. no defaulting.
```
Mode dbapi.MySQLGroupMode `json:"mode,omitempty"`
```